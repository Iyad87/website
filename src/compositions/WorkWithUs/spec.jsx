import React from 'react';
import { shallow } from 'enzyme';
import WorkWithUs from './index';

test('compositions/WorkWithUs snapshot', () => {
  const wrapper = shallow(<WorkWithUs />);

  expect(wrapper).toMatchSnapshot();
});
