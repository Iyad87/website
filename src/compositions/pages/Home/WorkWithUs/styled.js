import styled from 'styled-components';

import Container from 'components/Container';
import DefaultLink from 'components/Link';

import { highlight, textLight } from 'styles/colours';
import { spacing } from 'styles/spacing';

export const Section = Container.withComponent('section').extend`
  margin-bottom: ${spacing(5)};
  margin-top: ${spacing(5)};
  text-align: center;
`;

export const Title = styled.h4`
  color: ${textLight};
  font-size: 2.25rem;
  font-weight: 500;
  margin: 0;
`;

export const Details = styled.p`
  color: ${highlight};
  font-size: 2.25rem;
  font-weight: 300;
  margin: 0;
`;

export const Link = styled(DefaultLink)`
  font-weight: 300;
`;
