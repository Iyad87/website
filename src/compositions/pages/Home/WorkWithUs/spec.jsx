import React from 'react';
import { shallow } from 'enzyme';
import WorkWithUs from './index';

test('compositions/homepage/WorkWithUs snapshot', () => {
  const wrapper = shallow(<WorkWithUs />);

  expect(wrapper).toMatchSnapshot();
});
