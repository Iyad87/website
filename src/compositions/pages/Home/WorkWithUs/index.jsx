import React from 'react';

import { Details, Link, Section, Title } from './styled';

const WorkWithUs = () => (
  <Section wide>
    <Title>Work with us</Title>
    <Details>
      <Link href="mailto:hello@yld.io">hello@yld.io</Link>
      {' / '}
      <Link href="tel:+44-203-514-4678">+44 (0) 203 514 4678</Link>
    </Details>
  </Section>
);

export default WorkWithUs;
