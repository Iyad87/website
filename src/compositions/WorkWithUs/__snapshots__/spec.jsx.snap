// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compositions/WorkWithUs snapshot 1`] = `
<Container
  wide={true}
>
  <styled__Title>
    Work with us
  </styled__Title>
  <styled__Details>
    <styled__Link
      href="mailto:hello@yld.io"
    >
      hello@yld.io
    </styled__Link>
     / 
    <styled__Link
      href="tel:+44-203-514-4678"
    >
      +44 (0) 203 514 4678
    </styled__Link>
  </styled__Details>
</Container>
`;
